flink-cdc-e2e-tests/flink-cdc-source-e2e-tests/src/test/java/com/ververica/cdc/connectors/tests/MongoE2eITCase.java

```diff
@@ -122,6 +123,17 @@ public void after() {
         }
     }
 
+    @AfterClass
+    public static void afterClass() {
+        // Cleanup the mongo image, because it's too large and will cause the next test to fail.
+        container
+                .getDockerClient()
+                .listImagesCmd()
+                .withImageNameFilter("mongo:6.0.9")
+                .exec()
+                .forEach(image -> container.getDockerClient().removeImageCmd(image.getId()).exec());
```

Comment:
Thanks for you contribution, @gong , each CI will down image again after this change ?

Reply:
> Thanks for you contribution, @gong , each CI will down image again after this change ?

@leonardBang I think it also will download image if I don't add it. It will free space for other e2e test.